### PR TITLE
Feature/498 sandbox feedback

### PR DESCRIPTION
**Pull request by esotrope**
image

image

image

### CODE REVIEW

File: webapp/src/main/webapp/src/app/sandbox/sandbox-login.component.html [hunk 1]
```diff
@@ -1,68 +1,101 @@
...
-          type="text"
-          formControlName="username"
-          class="form-control"
-        />
-      </div>
-      <div>
-        <label for="role" class="control-label">
-          {{ 'sandbox-login.labels.role' | translate }}</label
+<ng-container *ngIf="!initialized; else loaded">
+  <spinner></spinner>
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/sandbox/sandbox-login.component.html]: page now has loading spinner when waiting for sandbox data

File: webapp/src/main/webapp/src/app/sandbox/sandbox-login.component.less [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  h1 {
+    margin: 15px 0 30px 0;
+    text-align: center;
+  }
+
+  button[type='submit'] {
+    margin-top: 20px;
+    text-align: center;
+  }
+}
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/sandbox/sandbox-login.component.less]: component styles > global / bootstrap utility classes

File: webapp/src/main/webapp/src/app/shared/form/information-button.component.html [hunk 3]
```diff
@@ -1,6 +1,5 @@
 <ng-container *ngIf="_title">
   <span innerHTML="{{ _title }}"></span>
-  &nbsp;
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/shared/form/information-button.component.html]: this was an accident

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 4]
```diff
@@ -1232,20 +1232,27 @@
...
       "Teacher": "Teacher: {{organizationName}} {{grade}}"
-    }
+    },
+    "sandbox-role-username": {
+      "DistrictAdministrator": "District Admin",
+      "SchoolAdministrator": "School Admin",
+      "Teacher": "Teacher"
+    },
+    "sandbox-info": "TODO",
+    "sandbox-role-info": "TODO"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: just left these todo @malaffoon

File: webapp/src/main/webapp/src/app/sandbox/sandbox-login.component.html (hunk 1 above)
Comment (malaffoon) [reply webapp/src/main/webapp/src/app/sandbox/sandbox-login.component.html]: make sure you take credit for RP-482

Comment (esotrope) [reply webapp/src/main/webapp/src/app/sandbox/sandbox-login.component.html]: done - thanks Mark

**Pull request merged**